src/full_node/full_node.py

```diff
@@ -645,145 +804,15 @@ async def respond_sub_block(
             raise ConsensusError(error_code, header_hash)
 
         elif added == ReceiveBlockResult.DISCONNECTED_BLOCK:
-            self.log.info(f"Disconnected block {header_hash} at height {sub_block.sub_block_height}")
-            peak = self.blockchain.get_peak()
-            if peak is None:
-                peak_height = -1
-            else:
-                peak_height = peak.sub_block_height
-            if sub_block.sub_block_height > peak_height + self.config["sync_blocks_behind_threshold"]:
-                if peak_height > sub_block.sub_block_height - self.constants.WEIGHT_PROOF_RECENT_BLOCKS:
-                    msg = Message(
-                        "request_sub_block",
-                        full_node_protocol.RequestSubBlock(uint32(sub_block.sub_block_height - 1), True),
-                    )
-                    self.full_node_store.add_disconnected_block(sub_block)
-                    if peer is not None:
-                        await peer.send_message(msg)
-                    else:
-                        return msg
-                else:
-                    async with self.blockchain.lock:
-                        if self.sync_store.get_sync_mode():
-                            return None
-                        self.sync_store.set_sync_mode(True)
-                    self.log.info(
-                        f"We are too far behind this block. Our height is {peak_height} and block is at "
-                        f"{sub_block.sub_block_height}"
-                    )
-                    # Performs sync, and catch exceptions so we don't close the connection
-                    self._sync_task = asyncio.create_task(self._sync())
-
-            elif sub_block.sub_block_height >= peak_height - 5:
-                # Allows shallow reorgs by simply requesting the previous height repeatedly
-                # TODO: replace with fetching multiple blocks at once
-                self.log.info(
-                    f"We have received a disconnected block at height {sub_block.sub_block_height}, "
-                    f"current peak is {peak_height}"
-                )
-                msg = Message(
-                    "request_sub_block",
-                    full_node_protocol.RequestSubBlock(uint32(sub_block.sub_block_height - 1), True),
-                )
-                self.full_node_store.add_disconnected_block(sub_block)
-                if peer is not None:
-                    await peer.send_message(msg)
-                else:
-                    return msg
+            self.log.warning(f"Disconnected block {header_hash} at height {sub_block.sub_block_height}")
```

Comment:
This seems pretty common. Unless I'm missing something about your changes it should probably be log.info

Reply:
Yeah makes sense, when there are network delays etc it could happen